### PR TITLE
pcengines-seabios-24.05.00.01: add keys

### PR DESCRIPTION
**Pull request by pietrushnic**

### CODE REVIEW

File: dasharo/pcengines_apu2/dasharo-release-24.05.00.x-for-pc-engines-signing-key.asc
Comment (miczyg1) [inline dasharo/pcengines_apu2/dasharo-release-24.05.00.x-for-pc-engines-signing-key.asc:1]: @pietrushnic why is it binary? IIRC all keys on the repo are armored

**Review by miczyg1 — CHANGES_REQUESTED** [review]

File: dasharo/pcengines_apu2/dasharo-release-24.05.00.x-for-pc-engines-signing-key.asc
Comment (pietrushnic) [reply dasharo/pcengines_apu2/dasharo-release-24.05.00.x-for-pc-engines-signing-key.asc:1]: oops, aparently wrong params during export, let me fix that

Comment (pietrushnic) [reply dasharo/pcengines_apu2/dasharo-release-24.05.00.x-for-pc-engines-signing-key.asc:1]: @miczyg1 force pushed the fix: https://github.com/3mdeb/3mdeb-secpack/pull/140/commits/9a764b80172e18b0bac46aaabd558c195a3d0d83

BTW, we could probably add such a check into pre-commit. Not sure how to do that or where to report, @macpijan ?

Comment (macpijan) [reply dasharo/pcengines_apu2/dasharo-release-24.05.00.x-for-pc-engines-signing-key.asc:1]: Not sure what the problem was exactly, but if you have two cases: "good" and "bad" you can make a script that exits with non-zero exit code on "bad" key file and add it as a local hook. Example: https://github.com/3mdeb/conferences/blob/master/.pre-commit-config.yaml#L51

Comment (miczyg1) [reply dasharo/pcengines_apu2/dasharo-release-24.05.00.x-for-pc-engines-signing-key.asc:1]: Checking for `-----BEGIN PGP PUBLIC KEY BLOCK-----` presence should be sufficient.

But also I would add a presence check for `-----BEGIN PGP PRIVATE KEY BLOCK-----` and fail if found. Not sure though how we can prevent adding private binary keys to the local repo... Maybe there are some utilities to do so as well.

**Review by miczyg1 — APPROVED** [review]

File: dasharo/pcengines_apu2/dasharo-release-24.05.00.x-for-pc-engines-signing-key.asc
Comment (macpijan) [reply dasharo/pcengines_apu2/dasharo-release-24.05.00.x-for-pc-engines-signing-key.asc:1]: This is already solved: https://github.com/3mdeb/3mdeb-secpack/blob/master/.pre-commit-config.yaml#L21

Comment (pietrushnic) [reply dasharo/pcengines_apu2/dasharo-release-24.05.00.x-for-pc-engines-signing-key.asc:1]: created issue for that, hopefully someone will have time to implement it at some point: https://github.com/3mdeb/3mdeb-secpack/issues/141

Comment (pietrushnic) [reply dasharo/pcengines_apu2/dasharo-release-24.05.00.x-for-pc-engines-signing-key.asc:1]: ideally if roadmap would be pushed every time we touching this repo

Comment (pietrushnic) [reply dasharo/pcengines_apu2/dasharo-release-24.05.00.x-for-pc-engines-signing-key.asc:1]: @miczyg1 please resolve if you think thread is done and merge

**Pull request merged**